WordPress/Resources/ar.lproj/InfoPlist.strings

```diff
@@ -1,5 +1,6 @@
-NSLocationUsageDescription = "WordPress would like to add your location to posts on sites where you have enabled geotagging.";
-NSLocationWhenInUseUsageDescription = "WordPress would like to add your location to posts on sites where you have enabled geotagging.";
-NSCameraUsageDescription = "لالتقاط صور أو مقاطع الفيديو لاستخدامها في مقالاتك.";
-NSPhotoLibraryUsageDescription = "لإضافة صور أو مقاطع فيديو إلى مقالاتك.";
-NSMicrophoneUsageDescription = "بالنسبة إلى مقاطع الفيديو المراد إضافة صوت إليها.";
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<!--Warning: Auto-generated file, do not edit.-->
+<plist version="1.0">
+  <dict/>
+</plist>
```

Comment:
See https://github.com/wordpress-mobile/WordPress-iOS/pull/18061 for why this and all the other `.strings` in this diff changed in such a way.